wg-serving/charter.md

```diff
@@ -0,0 +1,87 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for AI/ML inference workloads in Kubernetes.
+
```

Comment:
We should add @smarterclayton comments during the first call, on what's a Serving workload here in the charter.

Reply:
Yes please!

Reply:
+1 it overlaps a bit with the batch charter as written.

Reply:
Added one paragraph about it